### PR TITLE
update outbound ports

### PR DESCRIPTION
**Pull request by qiliRedHat**
https://issues.redhat.com/browse/PERFSCALE-3360

### CODE REVIEW

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml [hunk 1]
```diff
@@ -34,12 +34,13 @@ tests:
       CUSTOM_AZURE_REGION: centralus
       PROFILE_TYPE: reporting
       SET_ENV_BY_PLATFORM: custom
-      WORKER_REPLICA_COUNT: "120"
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml:37]: Please update the job name from `control-plane-ipsec-120nodes` --> `control-plane-ipsec-60nodes`

Comment (qiliRedHat) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml:37]: @krishvoor thanks, I'll change it back when the tests are done, using 60 to save some time and resource at early phase.

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55867/rehearse-55867-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.17-nightly-x86-control-plane-ipsec-120nodes/1828416364277141504
The step of updating outbount ports form 1024 to 46 passed and workers are scaled to 120 nodes. But the cdv-2 test passed because it exeeded 5 hours timeout, which is tracked in another Jira and I will open a separate PR to address it.
I think we can merge this PR now. @krishvoor @vishnuchalla PTAL

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.16-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.16-nightly-x86-control-plane-ipsec-120nodes

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-azure-4.16-nightly-x86-control-plane-ipsec-120nodes

File: ci-operator/step-registry/openshift-qe/installer/azure/update-allocated-outbound-ports/openshift-qe-installer-azure-update-allocated-outbound-ports-commands.sh [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    else
+        echo "Unexpected cluster: ${CLUSTER_TYPE}"
+        exit 1
+    fi
+
+    echo "$(date -u --rfc-3339=seconds) - Logging in to Azure..."
+    az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}" --output none || return 1
+}
+
+function gatherLBs() {
```
Comment (vishnuchalla) [inline ci-operator/step-registry/openshift-qe/installer/azure/update-allocated-outbound-ports/openshift-qe-installer-azure-update-allocated-outbound-ports-commands.sh:51]: Where is this function being used?

Comment (qiliRedHat) [reply ci-operator/step-registry/openshift-qe/installer/azure/update-allocated-outbound-ports/openshift-qe-installer-azure-update-allocated-outbound-ports-commands.sh:51]: @vishnuchalla It is not being used in the script right now. I copied it from another script.I think we may need to use it in the later when we have more platform to support like azurestack in that func.

Comment (qiliRedHat) [reply ci-operator/step-registry/openshift-qe/installer/azure/update-allocated-outbound-ports/openshift-qe-installer-azure-update-allocated-outbound-ports-commands.sh:51]: @vishnuchalla Let me know this looks good to you and we can go ahead to merge this PR.

Comment (vishnuchalla) [reply ci-operator/step-registry/openshift-qe/installer/azure/update-allocated-outbound-ports/openshift-qe-installer-azure-update-allocated-outbound-ports-commands.sh:51]: Lets get it added only when it is actually being used. Not much in favor of keeping dead code.

Comment (qiliRedHat) [reply ci-operator/step-registry/openshift-qe/installer/azure/update-allocated-outbound-ports/openshift-qe-installer-azure-update-allocated-outbound-ports-commands.sh]: @vishnuchalla agree, take out function gatherLBs.

Comment (qiliRedHat) [reply ci-operator/step-registry/openshift-qe/installer/azure/update-allocated-outbound-ports/openshift-qe-installer-azure-update-allocated-outbound-ports-commands.sh]: @vishnuchalla  I will not run the job again as I just taking out a not used function. Let's get this merged.

**Pull request merged**